src/permit-helper.js

```diff
@@ -1,4 +1,4 @@
-const ethers = require('ethers');
+const { ethers } = require('hardhat');
```

Comment:
Use `ethers` library is more light rather than `hardhat`
In this way, `hardhat` could be removed from `dependencies` and add it to  `devDependencies` in future iterations

Reply:
This was changed in https://github.com/0xPolygonHermez/zkevm-contracts/commit/4be88790576949795292010fbf439db2d9f8672b

Reply:
i see this was done to use hte provider, but this makes it dififcult to interact otuside of the project, will change it ^^